src/test/java/fakes/UserValidatorTest.java

```diff
@@ -1,49 +1,103 @@
 package fakes;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
+import java.util.Locale;
+
 class UserValidatorTest {
 
     @Nested
     class isValidUsername{
 
         @Test
         void returnsTrueIfOnlyLetters(){
```

Comment:
Das hier ist ein Testfall. Dieser testet eine Methode: new UserValidator().isValidUsername(user)) und prüft ob diese Methode sich gemäss dem Testnamen verhaltet.
Der Algorithmus der Validierung gehört in die User Klasse.

Das gilt für alle Testfälle.